web/app.fluidity.money/app/components/ViewRewardModal.tsx

```diff
@@ -37,7 +37,7 @@ export const ViewRewardModal = ({
 }: IPropsConnectedWalletModal) => {
   const [modal, setModal] = useState<React.ReactPortal | null>(null);
   const { width } = useViewport();
-  const isMobile = width < 500;
+  const isMobile = width < 500 && width > 0;
```

Comment:
Why do we need to check for width >0?

Reply:
It's related to server side rendering and the hook reading the screen size as 0 initially then updating to the actual screen size. It's been causing components to render on first load for a split second then disappear. This stops that.